Dockerfile

```diff
@@ -580,6 +581,10 @@ RUN \
   --disable-debug \
   --disable-shared \
   --disable-ffplay \
+  --disable-manpages \
+  --disable-podpages \
```

Comment:
I have a project where i use some of the documentation from the docker image but not the manpages, don't if others might find it useful to have.

Reply:
should have it installed in share/man then. right now only html is rendered i believe with theses settings